ui/tests/acceptance/enterprise-sidebar-nav-test.js

```diff
@@ -43,13 +43,18 @@ module('Acceptance | Enterprise | sidebar navigation', function (hooks) {
 
     await click(link('Disaster Recovery'));
     assert.strictEqual(currentURL(), '/vault/replication/dr', 'Replication dr route renders');
+    // for some reason this link would locally cause the testing browser to navigate
+    // manually redirecting to dashboard resolves the issue
+    await visit('/vault');
```

Comment:
The test suite couldn't reliably find `await click(link('Back to main navigation'));` so opted to manually redirect here